Userland/Libraries/LibDiff/Hunks.h

```diff
@@ -12,25 +13,24 @@
 
 namespace Diff {
 
+struct Range {
+    size_t start_line { 0 };
+    size_t number_of_lines { 0 };
+};
+
 struct HunkLocation {
-    size_t original_start_line { 0 };
-    size_t original_length { 0 };
-    size_t target_start_line { 0 };
-    size_t target_length { 0 };
-
-    enum class LocationType {
-        Original,
-        Target,
-        Both
-    };
-    void apply_offset(size_t offset, LocationType);
+    Range old_range;
+    Range new_range;
+};
+
+struct Line {
+    char operation; // Addition '+', Removal '-', Context ' ', or Change '!'.
```

Comment:
This feels like we should have an `enum Operation` here, and a `constexpr u32 symbolForOperation(Operation)` function

Reply:
Sounds good, will do!

Reply:
Done!